BUILD.bazel

```diff
@@ -318,7 +318,8 @@ filegroup(
 
 filegroup(
     name = "aten_native_mkldnn_cpp",
-    srcs = glob(["aten/src/ATen/native/mkldnn/*.cpp"]),
+    srcs = glob(["aten/src/ATen/native/mkldnn/*.cpp"],
+                exclude = ["aten/src/ATen/native/mkldnn/LlgaTensorImpl.cpp"]),
```

Comment:
no need for this `exclude`

Reply:
Thanks, will remove it, now that the file's been moved.